examples/cluster/plot_face_ward_segmentation.py

```diff
@@ -23,20 +23,14 @@
 
 from sklearn.feature_extraction.image import grid_to_graph
 from sklearn.cluster import AgglomerativeClustering
-from sklearn.utils.testing import SkipTest
-from sklearn.utils.fixes import sp_version
-
-if sp_version < (0, 12):
-    raise SkipTest("Skipping because SciPy version earlier than 0.12.0 and "
-                   "thus does not include the scipy.misc.face() image.")
 
 
 ###############################################################################
 # Generate data
 try:
     face = sp.face(gray=True)
 except AttributeError:
-    # Newer versions of scipy have face in misc
+    # SciPy 0.16 and higher have face in misc
```

Comment:
Also I prefer using a math sign to make it more clear whether 0.16 is included or not, e.g. something like:

```
feature bla does not exist in Scipy < 0.13.2
```